keps/sig-ui/5008-kep-template/kep.yaml

```diff
@@ -1,37 +1,39 @@
-title: Kubernetes Desktop
+title: Move Headlamp to the SIG UI
 kep-number: 5008
 authors:
   - "@joaquimrocha"
 owning-sig: sig-ui
-participating-sigs:
+participating-sigs: sig-ui
```

Comment:
Can be empty, no need to repeat the owning-sig here, just `[]` will get you there.